docs/TuxAnimations.md

```diff
@@ -0,0 +1,208 @@
+
+    <Fly duration={2000} delay={1000}>
+        <h1>Hello World</h1>
+    </Fly>
+
+#####Easing Property
+The Easing property determines what motion path the animation takes when presented. Easing only accepts a string which can take any custom cubic beziar animation path you wish. For example:
+
+    <Fly duration={3000} delay={1000} easing="cubic-bezier(0.680, -0.550, 0.265, 1.550)">
+        <h1>Hello World</h1>
+    </Fly>
+
+The Easing property also accepts any of the Tux default easings as a string:
+
+    <Fly duration={3000} delay={1000} easing="easeInOutBack">
+        <h1>Hello World</h1>
+    </Fly>
+
+The Tux default easings and thier motion paths are illustrated below. The 'Linear' path is not shown, but it is purely a straight line motion path.
+
+![Easings](http://i.imgur.com/2XAGQgq.png "Easings")
+
+#####Custom Property
+The Custom property accepts an object as an input with any CSS properties you wish the animation to include in addition to the default CSS transitions inherent of the particular componenet. For Example, if you wanted the element you've wrapped with a Fly animation component to also animate to the color red and increase its font size to 14px, it's as easy as:
+
+    <Fly duration={3000} delay={1000} easing="easeOutInBack"
+        custom={{'color': 'red', 'font-size': '14px'}}>
+        <h1>Hello World</h1>
+    </Fly>
+
+
+Finally, let's see how the default animation Fly animation looks now that it has been customized via its 4 default properties:
+
+![Animation 2](http://cdn.makeagif.com/media/1-12-2015/9Hqmtk.gif)
+
+####4) Make Your Own Animation Components
+Requiring Tux Animations exposes a makeAnimation function which allows the creation of custom animation components. The makeAnimation function creates a custom component based on a passed in transition object and then wrpas that custom component in a ReactTransitionGroup. The makeAnimation function has one required parameter; <code>transtions</code> and one optional paramter; <code>customClassName</code>
+
+    makeAnimation(transitions, customClassName);
+
+#####Paramater - <code>transitions</code> - type: OBJECT
+The transitions parameter requires an object that contains the transitions states that create your custom animation. This transition object has 4 required key value pairs:
+
+1. 'enter': **Object**
+2. 'enter-active': **Object**
+3. 'leave': **Object**
+4. 'leave-active': **Object**
+
+The keys define the animation state and the values are objects that describe the desired CSS properties of the component when that animaiton state has been completed.
+
+#####Paramater - <code>customClassName</code> - type: STRING
+The customClassName parameter requires a string that gives your component a custom className property so that it can be accessed easily via CSS or Jquery etc.. This way when your component is converted to a ReactElement, it is still accesible via this customClassName property:
```

Comment:
Change this and all other instances of Jquery to jQuery.
Correct spelling of accessible.